.gitignore

```diff
@@ -22,4 +22,6 @@
 .idea/dataSources.xml
 .idea/sqlDataSources.xml
 .idea/dynamic.xml
-.idea/uiDesigner.xml
\ No newline at end of file
+.idea/uiDesigner.xml
+.idea/libraries/*
+
```

Comment:
maybe you can also add `.idea/libraries/Maven__org_mockito_mockito_all [...] .xml` mentioned in your other PR.